eventuate-common-jdbc/src/main/java/io/eventuate/common/jdbc/sqldialect/EventuateSqlDialect.java

```diff
@@ -2,6 +2,16 @@
 
 public interface EventuateSqlDialect extends EventuateSqlDialectOrder {
   boolean supports(String driver);
+
   String getCurrentTimeInMillisecondsExpression();
+
   String addLimitToSql(String sql, String limitExpression);
+
+  default String castToJson(String sqlPart) {
+    return sqlPart;
+  }
+
+  default String objectToString(Object object) {
+    return object.toString();
```

Comment:
Where is this called? No where else in this PR?
Also, accepting any object and calling toString doesn't seem right.

Reply:
It is used in cdc, in polling dao: https://github.com/eventuate-foundation/eventuate-cdc/blob/4cda654a694f0cf9a6251a255f7ac76d764d9598/eventuate-local-java-cdc-connector-polling/src/main/java/io/eventuate/local/polling/PollingDao.java#L167

And it is used for string objects. Maybe I will just rename it to stringObjectToString?